README.md

```diff
@@ -8,6 +8,19 @@ This repo contains:
 * Instructions on how to install Nx Cloud using Helm
 * Instructions on how to install Nx Cloud using kubectl. See [here](./no-helm/README.md).
 
+## An important note on updates!
```

Comment:
some of our users maintain their own fork of our Helm repo adapted to their internal platforms. Can we add a guide please for these users if they want to update to the latest Docker images?